src/app/auth/sign-in/_components/signin-form.tsx

```
@@ -0,0 +1,104 @@
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+} from "@/components/ui/form"
+import { Input } from "@/components/ui/input"
+import { Label } from "@/components/ui/label"
+
+// Inner component that uses useSearchParams
+function SigninFormWithCallback({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)